core/src/main/resources/struts-beans.xml

```diff
@@ -198,7 +198,7 @@
     <bean type="ognl.MethodAccessor" name="com.opensymphony.xwork2.util.CompoundRoot"
           class="com.opensymphony.xwork2.ognl.accessor.CompoundRootAccessor"/>
 
-    <bean class="org.apache.struts2.views.jsp.ui.OgnlTool"/>
+    <bean class="org.apache.struts2.views.jsp.ui.OgnlTool"/> <!-- Deprecated since 6.3.0 -->
```

Comment:
Maybe it's better to put the comment before the bean?

Reply:
Amended - I tend to favour the same line as sometimes comments can become detached from their intended line in highly concurrent teams